src/lib.rs

```diff
@@ -118,6 +121,237 @@ pub fn builder() -> Builder {
+                .long("ignore-functions")
+                .help("Do not generate bindings for functions or methods. This is useful when you only care about struct layouts."),
+            Arg::with_name("ignore-methods")
+                .long("ignore-methods")
+                .help("Do not generate bindings for methods."),
+            Arg::with_name("dynamic")
+                .short("l")
+                .long("link")
+                .help("Link to dynamic library.")
+                .takes_value(true)
+                .multiple(true),
+            Arg::with_name("no-unstable-rust")
+                .long("no-unstable-rust")
+                .help("Do not generate unstable Rust code."),
+            Arg::with_name("opaque-type")
+                .long("opaque-type")
+                .help("Mark a type as opaque.")
+                .value_name("type")
+                .takes_value(true)
+                .multiple(true),
+            Arg::with_name("output")
+                .short("o")
+                .long("output")
+                .help("Write Rust bindings to <output>.")
+                .takes_value(true),
+            Arg::with_name("raw-line")
+                .long("raw-line")
+                .help("Add a raw line of Rust code at the beginning of output.")
+                .takes_value(true)
+                .multiple(true),
+            Arg::with_name("static")
+                .long("static-link")
+                .help("Link to static library.")
+                .takes_value(true)
+                .multiple(true),
+            Arg::with_name("use-core")
+                .long("use-core")
+                .help("Use types from Rust core instead of std."),
+            Arg::with_name("use-msvc-mangling")
+                .long("use-msvc-mangling")
+                .help("MSVC C++ ABI mangling. DEPRECATED: Has no effect."),
+            Arg::with_name("whitelist-function")
+                .long("whitelist-function")
+                .help("Whitelist all the free-standing functions matching <regex>. Other non-whitelisted functions will not be generated.")
+                .value_name("regex")
+                .takes_value(true)
+                .multiple(true),
+            Arg::with_name("whitelist-type")
+                .long("whitelist-type")
+                .help("Whitelist the type. Other non-whitelisted types will not be generated.")
+                .value_name("type")
+                .takes_value(true)
+                .multiple(true),
+            Arg::with_name("whitelist-var")
+                .long("whitelist-var")
+                .help("Whitelist all the free-standing variables matching <regex>. Other non-whitelisted variables will not be generated.")
+                .value_name("regex")
+                .takes_value(true)
+                .multiple(true),
+        ]) // .args()
+        .get_matches_from(bindgen_args);
+
+    let mut builder = builder();
+
+    if let Some(header) = matches.value_of("header") {
```

Comment:
instead of `if let Some(..) { } else { }`, what do you think about:

```rust
let builder = match matches.value_of("header") {
    Some(h) => builder.header(h),
    None => return Err(..),
};
```

I don't feel too strongly about it, so feel free to leave as-is if you prefer.